shared/chat/conversation/input-area/normal/mention-input.js

```diff
@@ -1,39 +1,49 @@
 // @flow
+/* eslint-disable sort-keys */
```

Comment:
@akalin-keybase Most of the key orders are semantic so I thought this was worthwhile